modules/core/src/lib/attribute.js

```diff
@@ -295,6 +301,12 @@ export default class Attribute extends BaseAttribute {
   }
 
   // PRIVATE HELPER METHODS
+  _getVertexOffset(index, bufferLayout = this.bufferLayout) {
```

Comment:
Do we need the `bufferLayout = this.bufferLayout`? 

I sometimes have used such code myself by I do think it looks a little "iffy", mixing state and input parameters in a somewhat subtle way.